tests/unit/sync/diff/contentType.test.ts

```diff
@@ -0,0 +1,216 @@
+import { describe, expect, it } from "@jest/globals";
+
+import { makeContentTypeHandler } from "../../../../src/modules/sync/diff/contentType";
+import { ContentTypeSyncModel } from "../../../../src/modules/sync/types/fileContentModel";
+
+describe("makeContentTypeHandler", () => {
```

Comment:
I would add test for taxonomyGroup as we had the problem with infinite loop :)

Reply:
That's definitely a good idea 👍